system/gameModes/Sidewinder/sizeAdjustment.lst

```diff
@@ -2,88 +2,88 @@
 # $Id$
 #
 SIZENAME:Fine		ABB:F	BONUS:ITEMCOST|TYPE=Ammunition,TYPE=Armor,TYPE=Shield,TYPE=Weapon|0.5
-SIZENAME:Fine		BONUS:ITEMWEIGHT|TYPE=Ammunition,TYPE=Armor,TYPE=Shield,TYPE=Weapon|0.1	BONUS:ITEMWEIGHT|TYPE=Goods|0.25
-SIZENAME:Fine		BONUS:ACVALUE|TYPE.Armor,TYPE.Shield|0.5	BONUS:COMBAT|AC|8|TYPE=Size	BONUS:COMBAT|TOHIT|8|TYPE=SIZE	BONUS:COMBAT|TOHIT.GRAPPLE|-24|TYPE=Size
-SIZENAME:Fine		BONUS:ITEMCAPACITY|TYPE=Goods|0.25
-SIZENAME:Fine		BONUS:SKILL|Hide|16|TYPE=SIZE
-SIZENAME:Fine		BONUS:LOADMULT|TYPE=SIZE|0.125|PRELEGSGTEQ:4
-#
+SIZENAME:F			BONUS:ITEMWEIGHT|TYPE=Ammunition,TYPE=Armor,TYPE=Shield,TYPE=Weapon|0.1	BONUS:ITEMWEIGHT|TYPE=Goods|0.25
```

Comment:
I think we should be using one standard here, either ABB on first line, or on last line. A mixture of both for different game modes is confusing. I'd vote for ABB on first line as then if someone wants to add a line they can follow the pattern.